packages/graphql-modules/tests/di-providers.spec.ts

```diff
@@ -542,101 +541,6 @@ test('Operation scoped provider should be created once per GraphQL Operation', a
   expect(loadSpy).toHaveBeenCalledWith(1);
 });
 
-test('Operation scoped provider should be created once per GraphQL Operation (Apollo Server)', async () => {
```

Comment:
@ardatan why it's deleted?

Reply:
I removed it because we don't have `createSchemaForApollo` anymore but now I've added it back by using Apollo Server itself.